src/components/Select/Select.tsx

```diff
@@ -147,12 +147,12 @@ export const Select: FC<SelectProps> = React.forwardRef(
 
     const getSelectedOptionValues = (): SelectOption['value'][] => {
       return options
-        .filter((option: SelectOption) => option.selected)
-        .map((option: SelectOption) => option.value);
+        ?.filter((option: SelectOption) => option?.selected)
```

Comment:
by falling back to array above you probably don't need these optional chains from here down